# 57. 예외처리 이해하기 (try ~ except ~ finally)

try:
    print('안녕하세요.')
    print('param')
except :
    print("예외가 발생 했습니다!")
finally:
    print("무조건 실행하는  코드")